llarp/service/protocol.cpp

```diff
@@ -39,26 +39,22 @@ namespace llarp::service
   ProtocolMessage::decode_key(const llarp_buffer_t& k, llarp_buffer_t* buf)
   {
     bool read = false;
-    if (!BEncodeMaybeReadDictInt("a", proto, read, k, buf))
-      return false;
+    // if (!BEncodeMaybeReadDictInt("a", proto, read, k, buf))
+    //   return false;
```

Comment:
Is this to be replaced or to be removed?

Reply:
unsure, i thought we discussed this proto as a member variable being replaced or re-implemented in some way? also aren't these objects going the way of the dinosaurs like the other message objects

maybe i'm falsely remembering, will mention this tomorrow in standup